pontoon/base/models/user.py

```diff
@@ -282,11 +282,12 @@ def can_translate(self, locale, project):
 
 
 def is_new_contributor(self, locale):
-    """Return True if the user hasn't made contributions to the locale yet."""
+    """Return True if the user has made just 1 contribution to the locale."""
```

Comment:
Mild preference for using a different function name, as this is setting up a surprising definition of "new contributor" that works in the context where it's used, but is here declared for User in general.

Reply:
I thought about that, but couldn't come up with a better name (the name was already misleading with the old logic). Do you have a suggestion? `has_one_contribution`?

Reply:
`has_one_contribution` is better.